adminer.tf

```diff
@@ -35,13 +51,13 @@ data "aws_iam_policy_document" "adminer" {
 
     condition {
       test     = "StringEquals"
-      variable = "ec2:ResourceTag/elasticbeanstalk:application-name"
-      values   = [local.app_name]
+      variable = "aws:ResourceTag/Block"
```

Comment:
Have you confirmed that AWS tags the EC2 Instances with `Block`?

Reply:
From what I'm reading on the AWS docs, I believe we need a setting on the launch configuration to propagate tags.
```
{
  namespace = "aws:autoscaling:launchconfiguration"
  name = "LaunchTemplateTagPropagationEnabled"
  value = "true"
}
```

Here is what I'm reading: https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html#command-options-general-autoscalinglaunchconfiguration
(Scroll down on that pane to find `LaunchTemplateTagPropagationEnabled`)

Reply:
I think that's just for `Launch Templates` which are apparently a different EC2 entity. All of the tags exist on EC2 and the Auto Scaling Groups already.